src/main/java/com/ardoq/ExcelImport.java

```diff
@@ -69,6 +71,59 @@ public static void main(String[] args) throws IOException {
         }
     }
 
+    private static void verifyModel() {
+        ArdoqClient client = new ArdoqClient(host, token);
+        client.setLogLevel(RestAdapter.LogLevel.NONE);
```

Comment:
Ideally, setting the loglevel should be an option, and at least placed somewhere else, as it's hidden in verify model method now.